db/migrate/20180321153346_add_tags_to_courses.rb

```diff
@@ -0,0 +1,5 @@
+class AddTagsToCourses < ActiveRecord::Migration[5.1]
+  def change
+    add_column :courses, :tags, :string, array: true, default: [], null: false
```

Comment:
Can we index this column as well?

Reply:
Would that just mean adding an index as an integer value?

Reply:
Done